src/wpool.erl

```diff
@@ -68,6 +70,18 @@
 -export([stats/0, stats/1, get_workers/1]).
 -export([default_strategy/0]).
 
+-if(?OTP_RELEASE >= 25).
+
+-type gen_server_request_id() :: gen_server:request_id().
```

Comment:
Instead of doing this, I would recommend to just use `gen_server:request_id/0` everywhere and ignore the warnings for OTP24. This project was always very ruthless in terms of its _no-backwards-compatibility_  policy ("if you want to use an older OTP version, use an older wpool version", that's the rule… and that's why there is a `rel-3.1` branch).

Reply:
So we move minimum to OTP 25 in CI?

Reply:
327b58a

Reply:
Yeah, we remove 24 from CI… or ignore warnings in that version.